### ISSUE TITLE
library not found for -lJTSImageViewController

### ISSUE THREAD

**Issue by mak2014** [fangwei716/30DaysOfReactNative#1]
Hello Wei Fang, Thanks for this. 

I am having issue running it: 

ld: library not found for -lJTSImageViewController
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by fangwei716** [comment]
Hi mak2014, I tested and don't have the issue for -lJTSImageViewController.
I use -lJTSImageViewController in Day4, Ref: https://shift.infinite.red/beginner-s-guide-to-using-cocoapods-with-react-native-46cb4d372995#.z6ooi3o0t
If you have cocoapods installed as the Ref instructed, you shouldn't run into this issue. 
However, I did forget to put two packages in packages.json and fixed that program.
You can download the latest version. I just tested, it should be able to run now. Thanks.

**Comment by mak2014** [comment]
Thanks Fang. I have deleted and clone the updated repo. I have followed your installation steps. When I run `npm install` i get the below error

`npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.2.6
npm ERR! npm  v2.14.12
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react-native@0.21.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer gl-react-native@2.22.0 wants react-native@^0.22.0
npm ERR! peerinvalid Peer react-native-chart@0.1.14 wants react-native@>0.15.0
npm ERR! peerinvalid Peer react-native-gesture-password@0.1.4 wants react-native@>=0.14 || 0.17.0-rc || 0.18.0-rc || 0.19.0-rc || 0.20.0-rc
npm ERR! peerinvalid Peer react-native-quick-actions@0.1.3 wants react-native@0.*
npm ERR! peerinvalid Peer react-native-scrollable-tab-view@0.4.0 wants react-native@>=0.20.0`

any help. pls
